Apps/WebClient/src/featuretoggleconfig.json

```diff
@@ -67,6 +67,14 @@
                 "name": "healthVisit",
                 "enabled": false
             },
+            {
+                "name": "hospitalVisit",
+                "enabled": false
+            },
+            {
+                "name": "medication",
+                "enabled": false
+            },
```

Comment:
I think the medication change slipped in from a merge conflict & should be removed.